conf/ocp/logback.xml

```diff
@@ -0,0 +1,16 @@
+<included>
+    <contextName>java-webapp</contextName>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{35} - %msg %n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
+      <appender-ref ref="STDOUT" />
+    </root>
+    <logger name="dk.kb" level="DEBUG" />
```

Comment:
There is no `dk.kb`-code and there is another `logback.xml` file at the folder-level above, so should this be deleted?